app/gui/view/documentation/assets/styles.css

```diff
@@ -0,0 +1,218 @@
+/* Custom CSS rules used by the documentation panel. */
+
+/* Common parts. */
+
+:root {
+  --type-name-color: #9640da;
+  --module-name-color: #a239e2;
+  --methods-header-color: #1f71d3;
+  --method-name-color: #1f71d3;
+  --types-header-color: #1f71d3;
+  --examples-header-color: #6da85e;
+  --important-background-color: #edefe7;
+  --info-background-color: #e6f1f8;
+  --example-background-color: #e6f1f8;
+  --docs-background-color: #fcfeff;
+  --docs-text-color: #434343;
+  --tag-background-color: #f5f5f5;
+  --caption-background-color: #0077f6;
+}
+
+.enso-docs {
+  font-family: "M PLUS 1", DejaVuSansMonoBook, sans-serif;
+  font-size: 11.5px;
+  color: var(--docs-text-color);
+  background-color: var(--docs-background-color);
+  padding-left: 0.5rem;
+  padding-right: 0.5rem;
+  padding-bottom: 0.5rem;
+}
+
+/* It's better to use `.unordered-list` class here instead of `ul`, but it will break the keyed lists that a provided by the engine.
+ * We need a separate way to represent keyed lists without embedded HTML tags. */
+ul {
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+  list-style-position: inside;
+}
+
+ul li:before {
+  content: "•";
+  font-size: 13px;
+  font-weight: 700;
+  margin-right: 3px;
+}
+
+ul li.type-item:before {
+  color: var(--type-name-color);
+}
```

Comment:
You should never create rules that are so broad. We should scope them very carefully, so they are not added to anything else. In this example, instead of `ul`, this should always be `.enso-docs ul`.

Reply:
Done